src/main/java/apoc/temporal/TemporalProcedures.java

```diff
@@ -13,65 +14,68 @@
-        ISO_DATE_FORMAT.put("basic_week_date_time_no_millis", DateTimeFormatter.ofPattern("xxxx'W'wwe'T'HHmmssZ"));
-        ISO_DATE_FORMAT.put("date", DateTimeFormatter.ofPattern("yyyy-MM-dd"));
-        ISO_DATE_FORMAT.put("date_hour", DateTimeFormatter.ofPattern("yyyy-MM-dd'T'HH"));
-        ISO_DATE_FORMAT.put("date_hour_minute", DateTimeFormatter.ofPattern("yyyy-MM-dd'T'HH:mm"));
-        ISO_DATE_FORMAT.put("date_hour_minute_second", DateTimeFormatter.ofPattern("yyyy-MM-dd'T'HH:mm:ss"));
-        ISO_DATE_FORMAT.put("date_hour_minute_second_fraction", DateTimeFormatter.ofPattern("yyyy-MM-dd'T'HH:mm:ss.SSS"));
-        ISO_DATE_FORMAT.put("date_hour_minute_second_millis", DateTimeFormatter.ofPattern("yyyy-MM-dd'T'HH:mm:ss.SSS"));
-        ISO_DATE_FORMAT.put("date_time", DateTimeFormatter.ofPattern("yyyy-MM-dd'T'HH:mm:ss.SSSZZ"));
-        ISO_DATE_FORMAT.put("date_time_no_millis", DateTimeFormatter.ofPattern("yyy-MM-dd'T'HH:mm:ssZZ"));
-        ISO_DATE_FORMAT.put("hour", DateTimeFormatter.ofPattern("HH"));
-        ISO_DATE_FORMAT.put("hour_minute", DateTimeFormatter.ofPattern("HH:mm"));
-        ISO_DATE_FORMAT.put("hour_minute_second", DateTimeFormatter.ofPattern("HH:mm:ss"));
-        ISO_DATE_FORMAT.put("hour_minute_second_fraction", DateTimeFormatter.ofPattern("HH:mm:ss.SSS"));
-        ISO_DATE_FORMAT.put("hour_minute_second_millis", DateTimeFormatter.ofPattern("HH:mm:ss.SSS"));
-        ISO_DATE_FORMAT.put("ordinal_date", DateTimeFormatter.ofPattern("yyyy-DDD"));
-        ISO_DATE_FORMAT.put("ordinal_date_time", DateTimeFormatter.ofPattern("yyyy-DDD'T'HH:mm:ss.SSSZZ"));
-        ISO_DATE_FORMAT.put("ordinal_date_time_no_millis", DateTimeFormatter.ofPattern("yyyy-DDD'T'HH:mm:ssZZ"));
-        ISO_DATE_FORMAT.put("time", DateTimeFormatter.ofPattern("HH:mm:ss.SSSZZ"));
-        ISO_DATE_FORMAT.put("time_no_millis", DateTimeFormatter.ofPattern("HH:mm:ssZZ"));
-        ISO_DATE_FORMAT.put("t_time", DateTimeFormatter.ofPattern("'T'HH:mm:ss.SSSZZ"));
-        ISO_DATE_FORMAT.put("t_time_no_millis", DateTimeFormatter.ofPattern("'T'HH:mm:ssZZ"));
-        ISO_DATE_FORMAT.put("week_date", DateTimeFormatter.ofPattern("xxxx-'W'ww-e"));
-        ISO_DATE_FORMAT.put("week_date_time", DateTimeFormatter.ofPattern("xxxx-'W'ww-e'T'HH:mm:ss.SSSZZ"));
-        ISO_DATE_FORMAT.put("week_date_time_no_millis", DateTimeFormatter.ofPattern("xxxx-'W'ww-e'T'HH:mm:ssZZ"));
-        ISO_DATE_FORMAT.put("weekyear", DateTimeFormatter.ofPattern("xxxx"));
-        ISO_DATE_FORMAT.put("weekyear_week", DateTimeFormatter.ofPattern("xxxx-'W'ww"));
-        ISO_DATE_FORMAT.put("weekyear_week_day", DateTimeFormatter.ofPattern("xxxx-'W'ww-e"));
-        ISO_DATE_FORMAT.put("year", DateTimeFormatter.ofPattern("yyyy"));
-        ISO_DATE_FORMAT.put("year_month", DateTimeFormatter.ofPattern("yyyy-MM"));
-        ISO_DATE_FORMAT.put("year_month_day", DateTimeFormatter.ofPattern("yyyy-MM-dd"));
+    static {
```

Comment:
can we move this to a separate class and use Util.map() and/or a helper method to make this setup more readable.

Reply:
Ok